### ISSUE TITLE
Run in KITTI dataset

### ISSUE THREAD

**Issue by crankler** [yunxdai/Intensity-based_Sidewalk_Segmentation#1]
Hi, I run this code in KITTI odom dataset 09, and I find the intensity-based mathod can't get a robust curb, can you tell me how to change teh yaml here? Thank you for your share

**Comment by yunxdai** [comment]
> Hi, I run this code in KITTI odom dataset 09, and I find the intensity-based mathod can't get a robust curb, can you tell me how to change teh yaml here? Thank you for your share

Hi, thank you for following this repo but I'm sorry this repo is only for idea validation, so it is not well structured and will no longer be maintained. If you are looking for curb detection methods, I recommend this [repo](https://github.com/wangguojun2018/LidarRoadBoundaryDetection), which I have successfully built and run on my own platform.

**Comment by crankler** [comment]
Yes, I have used part code of this pkg to detect the feature(just as the 3 kinds of judegment of his paper showed), I try a few param to test, but I stll find it's not good in my KITTI odom dataset,  I think maybe it's the result of moving objects.  So have you test this recommend rpk in KITTI dataset?  Also can you show me some ideas? 
thank you for your reply here.
